samples/Nancy.Demo.Razor.Localization/CustomResourceAssemblyProvider.cs

```diff
@@ -13,16 +11,22 @@
     /// </summary>
     public class CustomResourceAssemblyProvider : IResourceAssemblyProvider
     {
+        private readonly IAssemblyCatalog assemblyCatalog;
         private IEnumerable<Assembly> filteredAssemblies;
 
+        public CustomResourceAssemblyProvider(IAssemblyCatalog assemblyCatalog)
+        {
+            this.assemblyCatalog = assemblyCatalog;
+        }
+
         public IEnumerable<Assembly> GetAssembliesToScan()
         {
             return (this.filteredAssemblies ?? (this.filteredAssemblies = GetFilteredAssemblies()));
         }
 
-        private static IEnumerable<Assembly> GetFilteredAssemblies()
+        private IEnumerable<Assembly> GetFilteredAssemblies()
         {
-            return AppDomainAssemblyTypeScanner.Assemblies.Where(x => !x.IsDynamic);
+            return this.assemblyCatalog.GetAssemblies().Where(x => !x.IsDynamic);

```

Comment:
Also, the `.Where(x => !x.IsDynamic)` call is redundant; https://github.com/NancyFx/Nancy/blob/fade252d9aa9c25fe190706fb7ae6b66b0da675c/src/Nancy/AppDomainAssemblyCatalog.cs#L30